### DISCUSSION TITLE
GraphicsMagick?

### DISCUSSION THREAD

**Discussion by shockergit** [d2phap/ImageGlass#1270] [Q&A] [answered]
Does using GraphicsMagick as base increase the speed of ImageGlass viewing experience. I heard GraphicsMagick is faster than the orginal ImageMagick and also its well maintained?

**Comment by fire-eggs** [comment, marked as answer]
I replied to a similar suggestion in the Discord. ImageGlass uses Magick.NET to access ImageMagick. There would need to be a similar "GraphicsMagick.NET" component available to allow IG to use GraphicsMagick.

There is an old, archived repository called "GraphicsMagick.NET" but it is 5 years out-of-date. 

In addition, GraphicsMagick doesn't provide the same API as ImageMagick. So even if there was an up-to-date GraphicsMagick.NET, it could be non-trivial to modify IG to use the GraphicsMagick API.
